devApps/AdalDesktopTestApp/Program.cs

```diff
@@ -77,9 +77,11 @@ private static async Task RunAppAsync(AuthenticationContext context)
                         1. Clear the cache
                         2. Acquire Token by Integrated Windows Auth
                         3. Acquire Token Interactively
-                        4. Acquire Token with Username and Password
-                        5. Acquire Token Silently
-                        6. Acquire Token by Device Code 
+                        4. Acquire Token Interactively hidden
```

Comment:
>hidden [](start = 55, length = 6)

what does "hidden" mean?

Reply:
PromptBehavior.Never - i updated the code.